api/models/Story.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+const StorySchema = new mongoose.Schema({
+    author : { type : Schema.Types.ObjectId, ref : 'User' },
+    story: {type: String, required: true},
+    created: { type: Date, default: Date.now },
```

Comment:
At line 6 `Date.now`, should have this `()` ? right?

Reply:
It can be, but from the example that I took there was no ()... and it was a working example...